src/common/bit_str.h

```diff
@@ -15,6 +15,9 @@
 #define CEPH_COMMON_BIT_STR_H
 
 #include <functional>
+#include <ostream>
```

Comment:
this line is not necessary anymore. and could you order these includes in alphabetical order?

Reply:
also, please s/gcc-10/common,rbd/ in the title of the commit message. gcc-10 is not referencing any part of the source tree.

Reply:
this change is included by #32485 as well. as it turns out that they are also needed by clang-9 and libc++.